### ISSUE TITLE
lingui adding new characters into translations during extract | bug

### ISSUE THREAD

**Issue by seeden** [lingui/js-lingui#1051]
I have .po file with next context

```po
#: src/components/farm/FarmYourHarvesterNetwork.tsx:58
msgid "A harvester is a service running on a machine where plot(s) are actually stored. A farmer and harvester talk to a full node to see the state of the chain. View your network of connected harvesters below Learn more"
msgstr "Harvester là một dịch vụ chạy trên một  máy chứa các tệp plot  . Một farmer và harvester tương tác với một nút đầy đủ để xác định trạng thái của chuỗi. Xem các harvester được kết nối của bạn dưới đây Tìm hiểu thêm"
```

When I use `lingui extract` it change content of the file to 
```po
#: src/components/farm/FarmYourHarvesterNetwork.tsx:58
msgid "A harvester is a service running on a machine where plot(s) are actually stored. A farmer and harvester talk to a full node to see the state of the chain. View your network of connected harvesters below Learn more"
msgstr "Harvester là một dịch vụ chạy trên một  máy chứa các tệp plot  . Một \bfarmer và harvester tương tác với một nút đầy đủ để xác định trạng thái của chuỗi. Xem các harvester được kết nối của bạn dưới đây Tìm hiểu thêm"

```

As you can see there is an addition *\b* character that was not in the original file.

Lingui also changes other characters as well and it is not able to parse it anymore after this operation.

Here is my .linguirc file
```json
{
  "catalogs": [{
    "path": "<rootDir>/src/locales/{locale}/messages",
    "include": ["<rootDir>/src"],
    "exclude": ["**/node_modules/**"]
  }],
  "locales": ["da-DK", "de-DE", "en-AU", "en-PT", "en-US", "es-ES", "fi-FI", "fr-FR", "it-IT", "ja-JP", "nl-NL", "pl-PL", "pt-BR", "pt-PT", "ro-RO", "ru-RU", "sk-SK", "sv-SE", "zh-TW", "zh-CN"],
  "format": "po",
  "sourceLocale": "en-US",
  "fallbackLocales": {
    default: "en-US"
  }
}

```

FYI:
It is not working with all the latest versions of lingui.
This issue is related to https://github.com/lingui/js-lingui/issues/1033 as well.

**Comment by seeden** [comment]
Please reopen this issue. It was not fixed

**Comment by Martin005** [comment]
@seeden Hello, since we have fixed #1033 by updating the Messageformat parser package, that might have fixed your issue as well. Could you please try if the issue still persists on the newest version `3.16.1`?
